MapboxNavigation.podspec

```diff
@@ -47,6 +47,7 @@ Pod::Spec.new do |s|
   s.dependency "MapboxMaps", "10.0.0-rc.3"
   s.dependency "Solar", "~> 2.1"
   s.dependency "MapboxSpeech-pre", "2.0.0-alpha.1"
+  s.dependency "MapboxGeocoder.swift", "~> 0.10.0"
```

Comment:
Can we can remove the dependency on MapboxGeocoder.swift for CocoaPods too?

Reply:
This is actually needed to be able to successfully build Mapbox Navigation. We also link `MapboxGeocoder` for SPM.